packages/tokens/bin/lib/generateTokens.ts

```diff
@@ -0,0 +1,35 @@
+import {appendFileSync, outputFileSync} from 'fs-extra';
+
+import {formatScss} from './formats/formatScss';
+import {formatSvg} from './formats/formatSvg';
+import {formatTs} from './formats/formatTs';
+import {TokenList} from './formats/token';
+
+export const generateTokens = (name: string, tokens: TokenList) => {
+    [
+        {filePath: `${name}.scss`, outputFormat: 'scss'},
+        {filePath: `${name}.ts`, outputFormat: 'ts'},
+        {outputFormat: 'svg'},
+    ].forEach(({filePath, outputFormat}) => {
+        if (outputFormat === 'ts') {
+            const output = formatTs(tokens);
+            if (output) {
+                outputFileSync('./src/' + filePath, output);
+                appendFileSync('./src/index.ts', `export * from './${name}';\n`);
+            }
+        } else if (outputFormat === 'scss') {
+            const output = formatScss(tokens);
+            if (output) {
+                outputFileSync('./scss/' + filePath, output);
+                appendFileSync('./scss/index.scss', `@import '${name}';\n`);
```

Comment:
CSS rules not ready yet? Because I can't find any versioned SCSS folder in your pull request

Reply:
The icon library doesn't translate into css rules for now, it only creates svg files

Reply:
Can't wait to see what it will give for the CSS